home-exercises/src/main/java/advancedtest/exercise2/Basket.java

```diff
@@ -0,0 +1,42 @@
+package advancedtest.exercise2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<T> {
+    private T items;
+    private int currentCount = 0;
```

Comment:
Šis mainīgais īsti nav nepieciešams, jo vari visu laiku pārbaudīt itemList.size()

Reply:
Pēdējā nodarbībā runājām, šis bija galvā palicis no generics piemēriem, taču nevajadzēja galu galā izmantot un arī izdzēst piemirsās